src/views/simulation/resultats.vue

```diff
@@ -179,6 +184,41 @@ export default {
     isEmpty(array) {
       return !array || array.length === 0
     },
+    sendShowStatistics({ droitsEligibles, droits }) {
```

Comment:
Note: j'ai repris les deux logiques avec `droitsEligibles` et `droits`, je creuse pour comprendre si y'a une diff

Reply:
Alors il semblerait que `this.droits` soit un "computed" sur les droitsEligibles (cf [ici](https://github.com/betagouv/aides-jeunes/blob/f9797e74938000c3c9272d9e0568345580614f39/src/mixins/resultats.js#L5-L7)) 

Donc je vais enlever les références direct à `droitsEligible` pour utiliser `droits` defini dans le mixin